build.yaml

```diff
@@ -928,14 +935,16 @@ job_type:
   # http://build.clusterhq.com/builders/flocker-docs
   run_sphinx:
     run_sphinx:
-      on_nodes_with_labels: 'aws-centos-7-T2Small'
+      on_nodes_with_labels: 'aws-ubuntu-trusty-T2Small'
```

Comment:
this is the reason the job has been failling so far, something is fishy with sphinx on centos boxes.
It always fails on Centos, but works without any changes on Ubuntu